docs/low-code-sql/fabrics/snowflake.md

```diff
@@ -0,0 +1,67 @@
+| **4 Username** - Add the username that Prophecy will use to connect to the Snowflake Warehouse.                                                                                                                                                                                                                                                                                        |
+| **5 Password** - Add the password that Prophecy will use to connect to the Snowflake Warehouse. These username/password credentials are encrypted for secure storage. Also, each Prophecy user will provide their own username/password credential upon login. Be sure these credentials are scoped appropriately; Prophecy respects the authorization granted to this Snowflake user. |
+| **6 Role** - Add the Snowflake role that Prophecy will use to read data and execute queries on the Snowflake Warehouse. The role matches the username/password provided above and should be scoped according to the permission set desired for Prophecy.                                                                                                                               |
+| **7 Warehouse** - Specify the Snowflake warehouse for this execution environment.                                                                                                                                                                                                                                                                                                      |
+| **8 Database** - Specify the Snowflake database for this execution environment.                                                                                                                                                                                                                                                                                                        |
+| **9 Continue** to the Connections step, which is a feature coming soon for Snowflake Fabrics. Then Click Complete.                                                                                                                                                                                                                                                                     |
+
+### Connections
+
+(Coming soon for Snowflake Fabrics.)
+
+### Secrets
+
+([Secrets](https://docs.prophecy.io/low-code-spark/secret-management/) are currently implemented for Spark fabrics and are coming soon for SQL Fabrics.)
+
+Now the Snowflake Fabric is complete! Once you’ve created a Fabric, it will appear on the Metadata page. Team Admins can manage their Team’s Fabrics by clicking into the Fabric from this Metadata page. Prophecy SQL Projects can connect to your new Snowflake SQL Fabric, read tables, and execute models.
+![FabricMetadata](./img/FabricMetadata.png)
+
+## Scheduling
```

Comment:
Here we mention that the Airflow Fabric is needed to schedule a Snowflake model. I can add this in the intro as well.